virtual-env.ipynb

```diff
@@ -26,78 +31,116 @@
-    "    ```sh\n",
-    "    conda env export -n myenv > /v3io/users/$V3IO_USERNAME/virtual_env/myenv.yaml\n",
-    "    ```\n",
-    "    You can use the `User` data mount to the running-user directory to shorten this command (see [Platform Data Containers](data-ingestion-and-preparation/README.ipynb#platform-data-containers)):\n",
-    "    ```sh\n",
-    "    conda env -n myenv export > /User/virtual_env/myenv.yaml\n",
-    "    ```\n",
+    "   Using PIP:\n",
+    "   ```sh\n",
+    "   pip install <package> [<package> ...]\n",
+    "   ```\n",
+    "   <br>Alternatively, using Conda:\n",
+    "   ```sh\n",
+    "   conda install <package> [<package> ...]\n",
+    "   ```\n",
+    "   \n",
+    "   <br>For example, the following command uses PIP to install the SciPy, pandas version 1.4.4, and TensorFlow version 2.9.3 packages for the \"myenv\" environment that we activated in the previous step:\n",
+    "   ```sh\n",
+    "   pip install scipy pandas==1.4.4 tensorflow==2.9.3\n",
+    "   ```\n",
+    "   <br>\n",
     "\n",
+    "5. <a id=\"conda-virt-env-setup-step-export-env\"></a>Export your new virtual environment to an environment file in a platform data container by running the following command.\n",
+    "   Replace `<container name>` with the name of a platform data container, `<directory path>` with an optional relative container-directories path, and `<environment name>` with the name of the environment that you created:\n",
+    "   ```sh\n",
+    "   conda env export -n <environment name> > /v3io/<container name>[/<directory path>]/<environment name>.yaml\n",
+    "   ```\n",
+    "   <br>It is recommended that you save the environment to a virtual-environments directory in your running-user home directory (**/v3io/users/&lt;running user&gt;**).\n",
+    "   \n",
+    "   For example, the following command creates a **/v3io/users/&lt;running user&gt;/virtual_env/myenv.yaml** file:\n",
+    "   ```sh\n",
+    "   conda env export -n myenv > /v3io/users/$V3IO_USERNAME/virtual_env/myenv.yaml\n",
+    "   ```\n",
+    "   \n",
+    "   <br>To shorten this command, you can use the `/User` data mount to the running-user directory (see [Platform Data Containers](data-ingestion-and-preparation/README.ipynb#platform-data-containers)):\n",
+    "   ```sh\n",
+    "   conda env -n myenv export > /User/virtual_env/myenv.yaml\n",
+    "   ```\n",
+    "   <br>\n",
     "\n",
-    "5.  <a id=\"conda-virt-env-setup-step-refresh-ui\"></a>Refresh the JupyterLab UI to apply your changes.\n",
-    "    After refreshing the UI, you should be able to see your new environment in the list of available kernels in Jupyter Notebook."
+    "6. <a id=\"conda-virt-env-setup-step-refresh-ui\"></a>Refresh the JupyterLab UI to apply your changes.<br>\n",
+    "   After refreshing the UI, you should be able to see your new environment in the list of available kernels in JupyterLab."
```

Comment:
" After refreshing the UI, you should see your new environment in the list of available kernels in the JupyterLab."